paperspace/cli.py

```diff
@@ -0,0 +1,208 @@
+import functools
+import json
+
+import click
+
+from paperspace import commands, constants
+
+
+class ChoiceType(click.Choice):
+    """Takes a string-keyed map and converts cli-provided parameter to corresponding value"""
+
+    def __init__(self, type_map, case_sensitive=True):
+        super(ChoiceType, self).__init__(tuple(type_map.keys()), case_sensitive=case_sensitive)
+        self.type_map = type_map
+
+    def convert(self, value, param, ctx):
+        value = super(ChoiceType, self).convert(value, param, ctx)
+        return self.type_map[value]
+
+
+EXPERIMENT_TYPES_MAP = {
+    "GRPC": constants.EXPERIMENT_TYPE_GRPC_MULTI_NODE_ID,
+    "MPI": constants.EXPERIMENT_TYPE_MPI_MULTI_NODE_ID,
+}
+
+
+def json_string(val):
+    """Wraps json.loads so the cli help shows proper option's type name instead of 'LOADS'"""
+    return json.loads(val)
+
+
+def del_if_value_is_none(d):
+    """Remove all elements with value == None"""
+    for key, val in list(d.items()):
+        if val is None:
+            del d[key]
+
+
+@click.group()
+def cli():
+    pass
+
+
+@cli.group()
+def experiments():
+    pass
+
+
+@experiments.group()
+def create():
+    pass
+
+
+def common_expepriments_create_options(f):
+    options = [
+        click.option(
+            "--experimentEnv",
+            "experimentEnv",
+            type=json_string,
+        ),
+        click.option(
+            "--clusterId",
+            "clusterId",
+            type=int,
+        ),
+        click.option(
+            "--artifactDirectory",
+            "artifactDirectory",
+        ),
+        click.option(
+            "--workingDirectory",
+            "workingDirectory",
+        ),
+        click.option(
+            "--workspaceUrl",
+            "workspaceUrl",
+        ),
+        click.option(
+            "--ports",
+            type=int,
+        ),
+        click.option(
+            "--name",
+            required=True,
+        ),
+
+    ]
+    return functools.reduce(lambda x, opt: opt(x), options, f)
+
+
+@create.command(name="multinode")
+@common_expepriments_create_options
+@click.option(
```

Comment:
Add option - experimentType - either experimentTypeId or experimentType is required. Check comment above for available choices.

Reply:
It's already there --experimentTypeId